networking/ingress_load_balancing/configuring_ingress_cluster_traffic/ingress-controller-dnsmgt.adoc

```diff
@@ -1,33 +1,20 @@
 :_mod-docs-content-type: ASSEMBLY
 [id="ingress-controller-dnsmgt"]
-= Configuring an Ingress Controller for manual DNS Management
+= Understanding DNS management policies
 include::_attributes/common-attributes.adoc[]
 :context: ingress-controller-dnsmgt
 
 toc::[]
 
-As a cluster administrator, when you create an Ingress Controller, the Operator manages the DNS records automatically. This has some limitations when the required DNS zone is different from the cluster DNS zone or when the DNS zone is hosted outside the cloud provider.
+When you configure an Ingress Controller, you use the dnsManagementPolicy parameter to specify whether the Operator should manage the DNS records for you automatically or if you will manage them manually.
```

Comment:
This may be unclear to the reader who is who. "The operator" is "cluster ingress operator" or if we want to not go into details "platform" or "OpneShift". While "you will manage them" doesn't explain the fact that the cluster administrator is "you". It's important because "you" can be a user without any privileges.